src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -117,6 +119,20 @@ class GrabbitConfiguration {
         )
     }
 
+    /**
+     * Sorts the {@code pathConfigurations} on the basis of path depth
```

Comment:
I'm still of the opinion that we should implement this by :

```
I think we need to strip the leading node (or '.'), and forward slash; then compare by counting the number of forward slashes within paths.
```

I the way it's implemented now may accomplish the goal of having a given parent path before child path(s), but I may not be smart enough to think of all the ways it could break.  Additionally, it may be misconstrued in the future for doing something it's not - like sorting based on path depth. In my opinion, counting directory levels is a bit more straight forward.